specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,21 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/Microsoft.EventHub/stable/2022-10"
```

Comment:
Why does this path go to the project root and then move one directory up?

Reply:
ah b/c the data-plane folder is at the same level as the project-root. -->
```
- schemaregistry
       - SchemaRegistry
           - main.tsp
       - data-plane
           - Microsoft.EventHub
               - stable
                   - 2022-10
                       - schemaregistry.json                
 ```            
Should it be on a different level?

Reply:
I dont have a strong opinion on this one really, was just curious

Reply:
ah, I saw that other tsp files were created with this folder structure, but I move these if we want to in the future!

Reply:
Ah sorry for the confusion, I was talking about the `project-root` variable itself not the project's structure. The structure you have right now is good.

Reply:
ohh, gotcha that makes sense!